sdk/go-sdk/test/runtime/integrate_test.sh

```diff
@@ -14,8 +14,65 @@
 # limitations under the License.
 #
 
-go build ./cmd/layotto
+# start storage systems, e.g. redis, zk, etcd
 nohup redis-server --port 6380 &
-nohup ./layotto start -c ./configs/config_redis.json &
+bash /usr/share/zookeeper/bin/zkServer.sh start
+nohup bash /usr/share/zookeeper/bin/zkCli.sh < zkCreateZnode.sh
+cd ..
+nohup etcd &
+
+# build and run Layotto
+cd layotto
+go build ./cmd/layotto
+nohup ./layotto start -c ./configs/config_integrate_test.json &
+
+# run integrate_test
 cd sdk/go-sdk/test/runtime
-go test -p 1 -v ./...
\ No newline at end of file
+go test -p 1 -v ./...
+
+# run demos
+cd ../../../../demo/configuration/common
+go build -o client
+names="etcd_config_demo"
+for key in ${names}; do
+    ./client -s $key
+done
+
+cd ../../state/common
+go build -o client
+names="redis_state_demo zookeeper_state_demo"
```

Comment:
@bxiiiiii @zhenjunMa Hi, I just found that there is a panic error in CI:
https://github.com/mosn/layotto/actions/runs/3262814696/jobs/5360658524
![image](https://user-images.githubusercontent.com/26001097/196395947-7fe9ce52-3eaf-4f48-99e2-6c95288b3499.png)

So there are two issues:
1. We should fix the panic error
2. The CI shouldn' report success if a test case failed.

Reply:
@seeflood 
1.This is because of the problem of creating the znode script path, before entering the corresponding directory to run the script.
2.I think The CI is that it is based on the final go test result, but the demo run later does not run in the go test, which will also cause the previous test results to be washed up.

Reply:
fixed